spec/turn_spec.rb

```diff
@@ -0,0 +1,51 @@
+require './lib/turn'
+require './lib/card'
+require 'rspec'
+
+RSpec.describe Card do 
+    it 'initializes a card' do
+        card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+        expect(card).to be_instance_of(Card)
+    end
+end
```

Comment:
Even though our Turn class relies on getting a card object passed to it, we do not need to test the creation of card objects in this spec, because we already have tested that a card can be created and has all of its attributes in the card_spec. So, for that reason, you can get rid of this describe block.